README.md

```diff
@@ -1,7 +1,12 @@
 ﻿# relativity-integration-test-helpers
-Open Source Community: Integration testing is a software testing methodology used to test individual software components or units of code to verify their interaction. These components are tested as a single group or organized in an iterative manner. That said, we have created Relativity Integration Test Helpers to assist you with writing good Integration Tests for your Relativity application. You can use this framework to test event handlers, agents or any workflow that combines agents and Eventhandlers. We will continue adding more helpers but in the mean time you should be able to create workspaces, create dbcontext, proxy and create documents with this framework.
+## Overview
+Open Source Community: Integration testing is a software testing methodology used to test individual software components or units of code to verify their interaction. These components are tested as a single group or organized in an iterative manner.
+Relativity Integration Test Helpers was created to assist with writing best practice Integration Tests for Relativity applications. The framework can be used to test event handlers, agents or any workflow that combines agents and event handlers.
 
-TestHelpers v8 release is only compatible with Ninebark (12.0) and above.  Any previous version of TestHelpers will only be compatible with Relativity until Prairie Smoke, which will remove RSAPI.
+The framework will currently enable you to to proxy and create documents, create workspaces and create dbcontext with more functionality to be added in the future.
+
+## Compatibility
+TestHelpers v8 release is only compatible with Ninebark (12.0) and above. Any previous version of TestHelpers will only be compatible with Relativity until Prairie Smoke {INSERT VERSION NUMBER), which will remove RSAPI.

```

Comment:
Should we just say 12.2 instead of INSERT VERSION NUMBER?

Reply:
The power of getting distracted while doing something! Fixed haha